app/src/main/AndroidManifest.xml

```diff
@@ -1,131 +1,193 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="org.gdg.frisbee.android" android:versionCode="27" android:versionName="1.0.10" android:installLocation="auto">
+  package="org.gdg.frisbee.android"
+  android:installLocation="auto">
```

Comment:
the documentation says other: internalOnly is the default http://developer.android.com/guide/topics/manifest/manifest-element.html#install